python/fledge/services/core/api/configuration.py

```diff
@@ -293,14 +278,20 @@ async def set_configuration_item(request):
                 if 'readonly' in storage_value_entry:
                     if storage_value_entry['readonly'] == 'true':
                         raise TypeError("Update not allowed for {} item_name as it has readonly attribute set".format(config_item))
-            await cf_mgr.set_category_item_value_entry(category_name, config_item, value)
+            request_details = request if hasattr(request, "user") else None
+            await cf_mgr.set_category_item_value_entry(category_name, config_item, value, request=request_details)
         else:
             await cf_mgr.set_optional_value_entry(category_name, config_item, list(found_optional.keys())[0], list(found_optional.values())[0])
     except ValueError as ex:
         raise web.HTTPNotFound(reason=ex) if not found_optional else web.HTTPBadRequest(reason=ex)
     except (TypeError, KeyError) as ex:
         raise web.HTTPBadRequest(reason=ex)
-
+    except Exception as ex:
+        msg = str(ex)
+        if 'Forbidden' in msg:
+            msg = "Insufficient access privileges to change the value for category."
```

Comment:
This is for category level only or individual config items as well?

Reply:
Also mention which category/ configuration item <?>.

Reply:
only on config item